doc/network.tex

```diff
@@ -135,118 +138,132 @@
+Typical network protocols describe simple information exchanges;
+the distributed nature of blockchain computation means
+that additional contextual information is available.
+Use of this context allows, for example, for validated store and forward
+which is an essential feature to contain the effect of potential malicious actions
+against the distributed system.
 
-Shelley is a first fully decentralised release of Ouroboros protocol.
+Shelley is a first fully decentralised release of Cardano system, implementing the Ouroboros protocol.
 
 This Chapter contains an overview of the content and scope of this document and aspects that
 are being discussed.
 
+\wip{
 \subparagraph{Software assurance}
-\wip{Software assurance}
+Software assurance
+}
 
 \subparagraph{Layered Protocols}
 Traditionally network protocols are presented as a stack of layers where
 upper layers build on services provided by lower layers and lower layers
-are agnostic of the implementation of the upper layers.
+are independent of the implementation of the upper layers.
 This concept of layers is misleading when discussing Ouroboros.
-It is {\em not} the case that the consensus (layer) is build on top of the network (layer).
+For example, it is {\em not} the case that the consensus (layer)
+is built on top of the network (layer).
 It is more appropriate to talk about a network component than a network layer.
-The network component provides services to the consensus component and vice versa.
-Both components rely on each other.
-The network component uses the consensus component validate
-information that it distributes to other nodes.
-This is essential to guard against certain kinds of DoS attacks.
+The network component provides services to the consensus component and vice versa;
+both components rely on each other.
+The network component uses the consensus component to validate
+information that it distributes to other nodes, which
+is essential to guard against certain kinds of DoS attacks.
 Existing peer-to-peer systems focus on a slightly different problem domain.
 For example, they do not consider the information validation issue
-or have worries \todo{(Neil : I could not read the word)}, such as 'eclipse' that to not
+or are concerned with issues such as 'eclipse attacks' that to not
 apply to the Ouroboros family of protocols.
 
-\missingfigure[figwidth=6cm]{Testing a long text string}
+%\missingfigure[figwidth=6cm]{Testing a long text string}
 
-\subparagraph{Performance of Peer to Peer Networks}
-In computer science, Byzantine Fault Tolerance is a property which states
-that a distributed algorithm works for the honest participants of the algorithm
-under the assumption that a certain percentage of the participants is indeed honest.
-A similar, but more informal property applies to peer to peer networks as well.
+\subparagraph{Performance of the Ouroboros Network}
+In computer science, Byzantine Fault Tolerance is a property of a distributed algorithm, which states
```

Comment:
Still not happy with the flow here - how about this as a replacement paragraph:

It is not reasonable to assume perfection for the inter-peer data exchanges, nor that those peers are operating error free or non-adversarially. In the Ouroboros setting, there are interesting differences from the usual Byzantine Fault Tolerance scenarios found in Computer Science, there is more scope for interference, yet there are stronger conditions on notion of consistency.